src/components/userUtils/userSettingsPage.tsx

```diff
@@ -148,10 +137,6 @@ export default function UserSettingsPage({
             </Box>
           </Box>
         </Box>
-        <ThemedButton
-          title="Save"
-          onPress={() => navigation.push("DevelopmentLinks")}
-        />

```

Comment:
why remove? are we pushing settings updates to the backend onchange? i was thinking save/cancel dialogue would be a good idea

Reply:
Would the button just be saving the changes for switches? Most apps usually just change the setting when you interact with a switch/checkbox. I assumed that when you flip a switch, you would want to update that setting and it would just make a backend call

Reply:
i like that better, updating review